app/src/androidTest/java/uk/gov/onelogin/login/WelcomeScreenKtTest.kt

```diff
@@ -37,11 +41,29 @@ class WelcomeScreenKtTest : TestCase() {
     @BindValue
     val featureFlags: FeatureFlags = mock()
 
+    @BindValue
+    val navHostController: NavHostController = mock()
```

Comment:
don't think you need this anymore?

Reply:
good spot!
fix here: [7206791](https://github.com/govuk-one-login/mobile-android-one-login-app/pull/85/commits/72067912c151b49f94cc6efac677c1148dd05e26)